src/js/overlay.js

```diff
@@ -146,6 +146,7 @@ Overlay.prototype.loadContent = function(callback) {
 Overlay.prototype.render = function() {
 	var wrapperEl = document.createElement('div');
 	wrapperEl.className = 'o-overlay';
+	wrapperEl.classList.add('o-overlay--' + this.id.replace(' ', '-'));
```

Comment:
It'd be good to detail what this `replace` is doing. Is it here to simply trim the string?